clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -234,14 +234,6 @@ const TcfOverlay: FunctionComponent<OverlayProps> = ({
     [experience]
   );
 
-  const handleUpdateDraftState = useCallback(
-    ({ newEnabledIds, modelType }: UpdateEnabledIds) => {
-      const updated = { ...draftIds, [modelType]: newEnabledIds };
-      setDraftIds(updated);
-    },
-    [draftIds]
-  );
```

Comment:
this is kind of a fun thing—`draftIds` is an object that looks something like

```js
{
  purposesConsent: [1,2,3],
  purposesLegint: [4,5,6]
  ...
}
```
and this function is perfect for updating one field at a time, since it can say "set purposesConsent to [1,2,3]". However, for the all on/all off buttons, I wanted to update _two_ fields at once. in other words, I wanted to say "set purposesConsent to [1,2,3] and purposesLegint to [4,5,6]". my first instinct was to just continue using this function but call it twice. BUT that doesn't work because setting react state is asynchronous! therefore the first call fires, but since we do a `{...draftIds, [modelType]: newEnabledIds }`, when the second call fires, `draftIds` is still the old state (not updated yet) and therefore it looks like the first state never fired! 

so we really want to be able to set both states at the same time. therefore, I now pass the entire `setDraftIds` down to `TcfTabs`. most things only want to set one thing at a time (so this function moved to `TcfTabs` but now the all on/all off buttons can set multiple at once)